### PR TITLE
Added Release Notes for Hazelcast Operator 5.13.0

### PR DESCRIPTION
**Pull request by cheels**

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -1,17 +1,29 @@
 = Release Notes
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/release-notes.adoc:2]:
```suggestion
:description: These release notes list any new features, enhancements, fixes and breaking changes that were made between version 5.12.0 and version 5.13.0 of Hazelcast Operator.

{description}
```

File: docs/modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+- Added support for Secrets in Custom Config (CN-1320)
+- Added parameters to JetJob CR (CN-1314)
+- Added custom environment variable support (CN-1303)
+- Added support for IAM roles for service accounts in AWS (CN-1273)
 
 == Changes
 
-- Added "thread name" to the logging pattern. https://github.com/hazelcast/hazelcast-platform-operator/pull/1117[#1117]
-- Added `DefaultJavaOptions` and appended these options to the `JAVA_OPTS` environment variable. https://github.com/hazelcast/hazelcast-platform-operator/pull/1114[#1114]
+- OS cluster creation logic removed, and now it is required to provide the `licenseKeySecretName` (CN-1283, CN-1282)
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/release-notes.adoc:13]:
```suggestion
- OS cluster creation logic removed, and it is now required to provide the `licenseKeySecretName` (CN-1283, CN-1282)
```

**Review by oliverhowell — APPROVED** [review]
2 minor suggestions but pre-approving...

Not sure how useful it is for user to see Jira codes when they can't see the content - I know they're useful internally but would suggest taking these out as we've done for the other products/docsets.

**Comment by cheels** [comment]
> Not sure how useful it is for user to see Jira codes when they can't see the content - I know they're useful internally but would suggest taking these out as we've done for the other products/docsets.

@oliverhowell But I thought it's necessary because MC is doing the same https://docs.hazelcast.com/management-center/5.5/release-notes/5-5-0

**Comment by oliverhowell** [comment]
I think I gave the same advice! Not a biggie for this release though, we can deal with properly when/if we update the release notes template/process...

**Pull request merged**